yt/frontends/ramses/fields.py

```diff
@@ -192,6 +198,35 @@ def _temperature(field, data):
         if rt_flag: # rt run
             self.create_rt_fields()
 
+        #Load magnetic fields
+        if ('gas','magnetic_field_x_left') in self:
+            self.create_magnetic_fields()
+
+    def create_magnetic_fields(self):
+        #Calculate cell-centred magnetic fields from face-centred
+        def mag_field(ax):
+            def _mag_field(field, data):
+                return (data['magnetic_field_%s_left' % ax]+data['magnetic_field_%s_right' % ax])/2
+            return _mag_field
+
+        for ax in self.ds.coordinates.axis_order:
+            self.add_field(('gas',"magnetic_field_%s" % ax),
+                           sampling_type='cell',
+                           function=mag_field(ax),
+                           units=self.ds.unit_system['magnetic_field_cgs'])
+
+        def _divB(field,data):
+            '''Calculate magnetic field divergence'''
+            out = np.zeros_like(data['magnetic_field_x_right'])
+            for ax in data.ds.coordinates.axis_order:
+                out += data['magnetic_field_%s_right' % ax] - data['magnetic_field_%s_left' % ax] 
+            return out/data['dx']
+
+        self.add_field(('gas','divB'),
```

Comment:
Is this consistent with yt naming convention for magnetic fields? For velocity, it is for example `('gas', 'velocity_divergence')` so I would have expected to see `('gas', 'B_divergence')` here?

Reply:
There is already a field called magnetic_field_magnitude, which is auto-generated once all the fields are correctly loaded, so I don't think we need B^2.

I'm agnostic about calling it magnetic_field_divergence or divB. I've always heard it refer to as divB in conversation, but obviously magnetic_field_divergence would be more consistent.

Reply:
I opted for consistency and renamed it.